src/index.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="styles/style.css" />
+    <title>otus-three-column-layout</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-wrapper">
+        <a href="#">Услуги</a>
+        <a href="#">Партнеры</a>
+        <a href="#">Оплата</a>
+      </div>
```

Comment:
С точки зрения семантики, списки желательно оформлять в виде UL/LI. Блоки с навигацией лучше "оборачивать" в специальный тэг NAV.
```suggestion
  <nav class="header-wrapper">
    <ul>
      <li><a href="#">Услуги</a></li>
      <li><a href="#">Партнеры</a></li>
      <li><a href="#">Оплата</a></li>
    </ul>
  </nav>
```